solr/solrj/src/java/org/apache/solr/common/params/CommonParams.java

```diff
@@ -277,9 +279,23 @@ public static EchoParamStyle get(String v) {
    * <p>Defaults to 'false' if not specified.
    *
    * @see #REQUEST_ID
+   * @see #DISABLE_REQUEST_ID_DEFAULT
+   * @deprecated this was replaced by the auto-generated trace ids
    */
+  @Deprecated(since = "9.4")
   String DISABLE_REQUEST_ID = "disableRequestId";
 
+  /**
+   * An opt-out flag to prevent the addition of {@link #REQUEST_ID} tracing on distributed queries
+   *
+   * <p>Defaults to 'false' if not specified.
+   *
+   * @see #DISABLE_REQUEST_ID
+   * @deprecated this was replaced by the auto-generated trace ids
+   */
+  @Deprecated(since = "9.4")
+  boolean DISABLE_REQUEST_ID_DEFAULT = Boolean.getBoolean("solr.disableRequestId");
```

Comment:
Admittedly I hate seeing values in the Params classes but this practice started long ago and I see you are attempting to match some loosely followed conventions.

Reply:
makes sense to not have this in the CommonParams. will move to the SearchHandler